src/run/gtimes/_rt.ts

```diff
@@ -0,0 +1,41 @@
+import { Client, MessageEmbed } from "discord.js";
+
+export function _rt(client: Client) {
+  client.on("messageCreate", async (message) => {
+    if (message.author.bot) return;
+    if (message.content.startsWith("RT:")) {
+      const regex = /https:\/\/discord.com\/channels\/\d+\/(\d+)\/(\d+)/;
+      const str = message.content;
+      const match = str.match(regex);
+      if (match === null) return;
+      const [, channelID, messageID] = match;
+      const quoteChannel = client.channels.cache.get(channelID);
+      if ([quoteChannel, quoteChannel].includes(undefined)) return;
```

Comment:
同じ変数を 2 回配列に入れてるけど, おそらくやりたい処理はこれのはず
```suggestion
      if (match.includes(null)) return;
```